apps/src/applab/applab.js

```diff
@@ -231,7 +246,7 @@ function adjustAppSizeStyles(container) {
               // For this scale factor...
               // set the max-height and max-width for the visualization
               childRules[k].style.cssText = "max-height: " +
-                  Applab.appHeight * scale + "px; max-width: " +
```

Comment:
This scaling only happens on the non-share page. We only end up with the full 480 on the share page, where we add our 34px of footer. On the non-share page we're showing the footerless version of applab (i.e. at 446), hence our use of the footerless height.